lib/proto.js

```diff
@@ -85,6 +86,19 @@ function child (bindings) {
   return instance
 }
 
+function childBindings () {
+  const chindings = this[chindingsSym]
+  var chindingsObject = (chindings.indexOf(',') === 0) ? `{${chindings.substr(1)}}` : `{${chindings}}`
```

Comment:
nitpick: it's not an object yet, it's a string, maybe call it, .. `chindingsStr` or.. `chindingsJSON`